styles.css

```diff
@@ -16,6 +17,10 @@
   text-decoration: none;
 }
 
+main {
+  background-color: #f7f7f9;
+}
+
```

Comment:
- [ ] Kindly make sure your `main` section covers all widths of the page. You can achieve that by making sure the body doesn't have any `padding` or `margin`.